README.md

```diff
@@ -247,6 +247,12 @@ Name of the cron service. Specify this to change the platform's default.
 
 - *Default*: 'USE_DEFAULTS'
 
+user_crontabs (hash)
```

Comment:
In it's current state it's not actually a module parameter and used in hiera only.

Reply:
This has been fixed.